docs/notebooks/diagnostics/line_integrated_diagnostic.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>Could do with a title noting that the plot is a projection to a plane perpendicular to the line between the source and detector.</p><p> </p><p>Eh, you know what we should do, actually? Add pyvista as at least a docs dependency. It's 2021, let's do 3D plotting already. I volunteer to do that and modernise our plots after I wrap up with my thesis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I'm not sure I follow this comment. I agree making that more clear is a good, but is a plot title the best way? Rather than putting it in the text?

Reply:
Ah, sorry. The plot title was just the first thing that came to mind. Some way of communicating "this situation is 3D, but the reason we're seeing a 2D plot is that we're projecting to the XZ (IIRC) plane" would be great :)